manifest/v1alpha/slo/metrics_sumo_logic_test.go

```diff
@@ -187,22 +187,78 @@ _collector="n9-dev-tooling-cluster" _source="logs"
   | sort by n9_time asc`,
 			Error: testutils.ExpectedError{
 				Prop:    "spec.objectives[0].rawMetric.query.sumoLogic.query",
-				Message: "exactly one timeslice declaration is required in the query",
+				Message: "query must contain a 'timeslice' operator",
```

Comment:
[nit] Do we want to put this message in a `const` and reuse it?

It's also used here:
https://github.com/nobl9/nobl9-go/pull/238/files#diff-77a902bb39ea318908cd500a58d5cf1454e6229f90b04fb53314b1d9e89e0eceR154